src/utils/errors.h

```diff
@@ -46,18 +30,12 @@ errorMessage const ERROR_MSG_JSON_KEY_NOT_FOUND = "Json does not contain key: ";
 errorMessage const ERROR_MSG_JSON_SET = "Json can not insert key:  ";
 errorMessage const ERROR_MSG_HTTP_REQUEST_FAILED = "Request failed with message: ";
 errorMessage const ERROR_MSG_REASON = "Error reason: ";
+errorMessage const ERROR_MSG_KEY_FILE = "Invalid keyfile.";
+errorMessage const ERROR_MSG_MAC = "MAC mismatch, possibly wrong password.";
+errorMessage const ERROR_MSG_SCRYPTSY = "Scrypt function failed. Could not derive keys, possible cause: operating system refused to allocate the amount of requested memory.";
```

Comment:
Is this the most probable cause? Why?

Reply:
This is from libsodium's documentation here: 
https://libsodium.gitbook.io/doc/advanced/scrypt#low-level-scrypt-api

Which states that: 

> The function returns 0 on success, and -1 if the computation didn't complete, usually because the operating system refused to allocate the amount of requested memory.